aait/backend/group-2B/Application.Tests/Mocks/Infrastructure/MockJwtGenerator.cs

```diff
@@ -0,0 +1,17 @@
+using Moq;
+using SocialSync.Application.Contracts.Infrastructure;
```

Comment:
Here, it would be appropriate to mock the shared dependency only. The JwtGenerator and PasswordHasher is not a shared dependency, so it should not be mocked.

Reply:
Understood. Will be fixed.